pkg/storage/storage_handler.go

```diff
@@ -0,0 +1,44 @@
+package storage
+
+import (
+	"github.com/cyberark/conjur-authn-k8s-client/pkg/storage/config"
+)
+
+type SecretsHandler interface {
+	HandleSecrets() error
+}
+
+type AccessTokenHandler interface {
+	Read()	([]byte, error)
+	Write()	error
+	Delete()	error
+}
+
+// Need underscore here unsure why
+type Storage_Handler struct {
+	AccessToken AccessTokenHandler
+	Secrets SecretsHandler
+}
+
+func NewStorageHandler(config config.Config) (StorageHandler *Storage_Handler, err error) {
+	// check what StoreType was in original store
+	if config.StoreType == "k8s_secrets" {
```

Comment:
don't we have a const for this?

Reply:
in a nested package. Do we want to add another const here? or is there a way to use the same const across different packages?

Reply:
did et